contrib/babelfishpg_tsql/sql/sys_function_helpers.sql

```diff
@@ -1819,7 +1819,7 @@ DECLARE
     HH_REGEXP CONSTANT VARCHAR COLLATE "C" := concat('^', TIMEUNIT_REGEXP, '$');
     DATATYPE_REGEXP CONSTANT VARCHAR COLLATE "C" := '^(TIME)\s*(?:\()?\s*((?:-)?\d+)?\s*(?:\))?$';
 BEGIN
-    v_datatype := trim(regexp_replace(p_datatype, 'DATETIME', 'TIME', 'gi'));
+    v_datatype := trim(regexp_replace(p_datatype COLLATE "C", 'DATETIME', 'TIME', 'gi'));
```

Comment:
Should we directly add collate in arg?

Reply:
We cant add default collation to input arguments